### PR TITLE
JMAP Email: destroying an already destroyed email should return notFound

### PR DESCRIPTION
**Pull request by ksmurchison**
Fixes an issue discovered by FM UI

### CODE REVIEW

**Review by brong — APPROVED** [review]
Yeah this looks good.

**Comment by ksmurchison** [comment]
@rsto I’m going to wait for your review before merging this.

**Review by rsto — APPROVED** [review]

**Pull request merged**
